english/forest/fire/woods/follow/shoreline/river/river.md

```diff
@@ -0,0 +1 @@
+You throw yourself back into the river and flow downstream. In the distance, you see a boat. You start swimming to the possibility of safety.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.